config/vufind/icons.ini

```diff
@@ -0,0 +1,16 @@
+[Config]
+use=FontAwesome
+
+[FontAwesome]
+favorites=Remix:star-fill
```

Comment:
I think this is an effective way of making things more functional, but I'm still not convinced that we want this to come from an .ini file. What if themes.config.php just included a section like:

<pre>
'icons' => [
    'defaultSet' => 'FontAwesome',
    'mappings' => [
        'favorites' => 'Remix:star-fill',
        'my-account' => 'user',
        'search' => 'Remix:search-line',
    ]
]
</pre>

As long as we set up the theme config loading so that it appropriately merges settings through the theme inheritance tree, this would make it possible to swap out individual icons or make larger-scale changes on a theme by theme basis. People could even publish mix-ins designed ONLY to change the icon system. It just feels to me more in keeping with how themes work. But again, happy to discuss further. :-)

Reply:
Can't argue with that! I've moved it to theme.config.php.